mavlink_messages_size.py

```diff
@@ -0,0 +1,134 @@
+from collections import OrderedDict
+
+mavlink_message_lengths_dict = OrderedDict([

```

Comment:
Should be able to get these message lengths with a little bit of introspection on pymavlink?